packages/button/Button.test.tsx

```diff
@@ -1,45 +1,64 @@
 import * as expect from 'expect';
-import { h } from '../_helpers';
-import { mount } from 'bore';
+import { h, mount } from 'bore';
+import { emit } from 'skatejs';
 
 import { Button } from './index';
 
 describe( Button.is, () => {
 
-describe( `Custom element`, () => {
-  it( `should be registered`, () => {
-    expect( customElements.get( Button.is ) ).toBe( Button );
-  });
-  it( `should render via JSX IntrinsicElement`, () => {
-    return mount( <bl-button>Hello</bl-button> ).wait( (element) => {
-      expect( element.node.localName ).toBe( Button.is );
+  describe( `Custom element`, () => {
+
+    it( `should be registered`, () => {
+      expect( customElements.get( Button.is ) ).toBe( Button );
     });
-  });
-  it( `should render`, () => {
-    return mount( <Button /> ).wait(( element ) => {
-      expect( element.has( '.c-button' ) ).toBe( true );
+
+    it( `should render via JSX IntrinsicElement`, () => {
+      return mount( <bl-button>Hello</bl-button> ).wait(( element ) => {
+        expect( element.node.localName ).toBe( Button.is );
+      });
+    });
+
+    it( `should render`, () => {
+      return mount( <Button /> ).wait(( element ) => {
```

Comment:
then please fix it to `<bl-button>`

thanks

p.s. this tone of suggestion is really unfortunate, point of PR is answering questions. PR comments are not the right place for discussing attitudes and any PR unrelated topic. Please contact me directly if you want to discuss this further. Thanks